.github/workflows/rebuilder.py

```diff
@@ -329,6 +377,20 @@ def main():
             project_owner, project_name, start_time, copr_pkgs
         )
         print(json.dumps(pkg_failures))
+    elif args.command == "get-snapshot-date":
+        project = copr_client.project_proxy.get(project_owner, project_name)
+        for repo in project['additional_repos']:
+            match = re.match(r"copr://@fedora-llvm-team/llvm-snapshots-big-merge-([0-9]+)$", repo)
+            if not match:
+                continue
+            print(datetime.datetime.fromisoformat(match.group(1)).isoformat())
```

Comment:
This prints more than one date so I suggest to rename it to `get-snapshot-dates`.